buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java

```diff
@@ -1239,6 +1243,9 @@ private void createConfiguration() {
         baseConfig.put("path.logs", confPathLogs.toAbsolutePath().toString());
         baseConfig.put("path.shared_data", workingDir.resolve("sharedData").toString());
         baseConfig.put("node.attr.testattr", "test");
+        if (this.project.findProperty("numZones") != null) {
```

Comment:
Why we need to check `numZones`? `if (zone != null && !zone.isBlank()) {` should be sufficient, right?

Reply:
Since we are passing zone value for the first node as well. Which will be true for the scenario where -PnumZones will not be set. Hence wanted to check the existence of the property.
Don't want to add zone attribute for the single node cluster. i.e 
``` ./gradlew run ```

Reply:
The decision should be made in `OpenSearchCluster`: this class knows zones configuration (and should either associated node with zone or not).

Reply:
Moved the decision making to OpensearchCluster class.

Reply:
TY